sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -56,68 +54,40 @@ add the direct dependency to your project as follows.
 ### Prerequisites
 
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
-- [Azure Subscription][azure_subscription]
-- An existing [Azure Key Vault][azure_keyvault]. If you need to create a Key Vault, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names:
+- An [Azure Subscription][azure_subscription].
+- An existing [Azure Key Vault][azure_keyvault]. If you need to create a Key Vault, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-key-vault-name>` and `<your-resource-group-name>` with your own, unique names:
```

Comment:
Client Id is not required. Default Azure Credential automatically picks identity in specified order. In case of local it takes it from CLI/PSH or VS login.